### PR TITLE
Feat:Login Screens & Css Misc

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
- Added Login UI
- Added Register UI
- Added CSS Util like Padding,Margin
- Made Form Resusable Component
- Made Modal Resusable Component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
 import { Routes, Route } from "react-router-dom";
 import Home from "./Pages/Home";
```
Comment (VedanthB) [inline src/App.js:2]: You can do a barrel import / export for all you components / files. Makes the code cleaner

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import Login from "./Pages/Login";
+import Register from "./Pages/Register";
 import "./App.scss";
 
 function App() {
   return (
     <div className="App">
       <Routes>
```
Comment (VedanthB) [inline src/App.js:10]: You can also move the routes into a diff file , make the code clean

**Review by VedanthB — APPROVED** [review]
Good work 

things i liked 
- clean organised code 
- use of sass

You can merge the PR

File: src/Components/Form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import Button from "../Button";
+
+const Index = ({ title, fields, onTextInput, submitCallBack }) => {
+  return (
+    <div className="form-container">
+      <div className="form-title">{title}</div>
+      {Object.keys(fields).map((key) => {
+        const { name, type, placeholder, tagName, required } = fields[key];
+        return (
+          <div class="kash-container  kash-flex kash-justify-center kash-align-center  kash-gap input-container">
```
Comment (ashishnnnnn) [inline src/Components/Form/index.jsx:12]: class can be replaced with ClassName

**Review by ashishnnnnn — APPROVED** [review]
Overall Code Seems Pretty Much Fine.. :+1:

**Pull request merged**